packages/block-library/package.json

```diff
@@ -44,6 +44,7 @@
 		"@wordpress/date": "file:../date",
 		"@wordpress/deprecated": "file:../deprecated",
 		"@wordpress/dom": "file:../dom",
+		"@wordpress/editor": "file:../editor",
```

Comment:
I don't think `@wordpress/editor` should be introduced as a dependency to `@wordpress/block-library`. I know it is used in a couple of places (calender, table of contents, template part), but I think introducing this as an explicit dependency was always avoided.

Reasoning - the dependency chain is conceptually the other way around (editor uses blocks), so it's a bit risky in terms of the chance of introducing a circular dependency.

Other blocks use the 'core/editor' string to avoid the package dependency and treat it as an optionally available store. There's also usually a comment about the problem.

Switching to `useEntityProp` would be good to explore though.

Reply:
By rewriting `getEditedPostAttribute` to `useEntityProp`, the import of this package is no longer necessary 👍